docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.4"
+
+volumes:
+  mongodb_data_container:
+
+services:
+  nightscout:
+    image: "skalahonza/nightscout"
```

Comment:
This seems to be obsolete and should be discarded.

Reply:
Good point. Discarded.